src/components/VideoGrid/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import VideoCard from '../VideoCard';
+import { Grid, Row, Col } from './styled';
+
+const VideoGrid = ({ videos }) => (
+  <Grid data-testid="grid">
+    <Row data-testid="row">
+      {videos.map((video) =>
+        video.id.videoId ? (
+          <Col key={video.id.videoId}>
+            <VideoCard key={video.id.videoId} video={video} />
+          </Col>
+        ) : (
+          ''
+        )
+      )}
```

Comment:
Could improve doing:
- using the provided index by map fn at the key prop for the `Col` component this case
- The second key prop is not necessary, it is used at Col only.
```suggestion
      {videos.map((video, index) =>
        video.id.videoId ? (
          <Col key={index}>
            <VideoCard video={video} />
          </Col>
        ) : (
          ''
        )
      )}
```